Code/Editor/Plugins/EditorAssetImporter/ImporterRootDisplay.h

```diff
@@ -40,25 +42,57 @@ namespace AZ
 
         }
     }
-}
+} // namespace AZ
```

Comment:
are these namespace end comments required?

Reply:
no, but I've found them useful

Reply:
We do call this out in the coding standards - https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#namespace-names (and `clang-format` will helpfully add them too)